core/src/entities/packages/unsupported.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
```

Comment:
@qtpeters  I wasn't planning to keep a whole `Unsupported` struct forever, just during initial debugging. That said, I do think it would be useful to have a log of things that weren't supported by the default SBOM provider. Any thoughts on where and how to log these things?  It might end up being possible to get an SBOM for them using a different provider.